### DISCUSSION TITLE
Project Dev guidance

### DISCUSSION THREAD

**Discussion by microtechno9000** [automatic-ripping-machine/automatic-ripping-machine#547] [Q&A] [answered]
I have been working on making some changes to ARM, but before submitting a PR want some info on the Dev rules or guidance for this project. Looking at the Dev wiki, it is a little light on contribution guidelines.

I have tried to keep function names and naming style similar to the existing functions and classes. Is ARM using a specific guide or styling rules? 
like https://peps.python.org/pep-0008 ??

**Comment by shitwolfymakes** [comment]
The current dev team inherited the code base from the original dev team, so we're in the process of making it our own as well as cleaning it up.

No formal guidelines exist at present, but PEP8 compliance is part of my personal expectations. I might even suggest adding `black` later, depending on the circumstances.

**Comment by 1337-server** [comment, marked as answer]
I use pep8 as the main guidelines, pylint and sonarlint is used for checks.

PRs have their own checks, so as long as it passes, it should be fine.

**Comment by MrTechGadget** [comment]
Thanks for everyone that has been contributing and evolving this project!
